src/components/PostCompletionViews/PostCompletionView.tsx

```diff
@@ -39,28 +49,21 @@ const PostCompletionView = ({
 
       {mainWebsiteUrl ? (
         <ButtonGroup className="utrecht-button-group--distanced" direction="column">
```

Comment:
Shouldn't this be `className="openforms-form-navigation" direction="column"` too?
There is one more place where `utrecht-button-group--distanced` is used.

Reply:
I thought so too initially, but this is not the usual pattern of the save link etc. and the button itself doesn't have an icon or anything, so I think it can stay as-is.